### PR TITLE
ADBP-2252:Bug fixed

### PR DESCRIPTION
**Pull request by KashifM101**
Link to Jira:
https://101digital.atlassian.net/browse/ADBP-2252

### CODE REVIEW

File: src/components/transfer-details-component/index.tsx [hunk 1]
```diff
@@ -52,20 +52,20 @@ export type RenderItemType = {
...
+    accoutType: 'Current Account-ii',
+    accountNumber: '104090703',
     amount: 5090,
   },
   {
     id: 3,
-    accoutType: "Fixed Deposit Account-iii",
-    accountNumber: "104090713",
+    accoutType: 'Fixed Deposit Account-iii',
+    accountNumber: '104090713',
```
Comment (sharmal33) [inline src/components/transfer-details-component/index.tsx:68]: why accountNumber are hardcoded ?

Comment (sharmal33) [reply src/components/transfer-details-component/index.tsx:68]: accoutType and accountNumber should return from API

**Review by sharmal33 — CHANGES_REQUESTED** [review]

File: src/components/transfer-details-component/index.tsx (hunk 1 above)
Comment (KashifM101) [reply src/components/transfer-details-component/index.tsx:68]: i don’t know anything about that @sharmal33 may be the one who had worked on it have the idea

**Review by khoi101digital — APPROVED** [review]

File: src/components/transfer-details-component/index.tsx (hunk 1 above)
Comment (sharmal33) [reply src/components/transfer-details-component/index.tsx:68]: can you ask this with @hiteshdigit101  
this values should return from api

Comment (KashifM101) [reply src/components/transfer-details-component/index.tsx:68]: ok

**Pull request merged**